test/langtools/jdk/javadoc/doclet/testParamTaglet/pkg/C.java

```diff
@@ -38,4 +38,39 @@ public void regularParams(int param1, int param2) {}
      */
     public void nonMatchingParams(int param1, int param2) {}
 
+    /**
+     * Overriding method with missing/additional/duplicate param tags in non-declaration order.
+     *
+     * @param x does not exist
+     * @param b a boolean
+     * @param i an int
+     * @param x duplicate
+     * @param b another duplicate
+     */
+    public void unorderedParams(int i, double d, boolean b) {}
+
+    /**
+     * Generic method with mixed/missing param tags.
+     *
+     * @param t1 param 1
+     * @param <T2> type 2
+     * @param t3 param 3
+     */
+    public static <T1, T2, T3> void genericMethod(T1 t1, T2 t2, T3 t3) {
```

Comment:
You use `{}` to denote an empty body everywhere else in this file. Is there a reason you split it into two lines here?